spec/javascript/controllers/talk-to-us_controller_spec.js

```diff
@@ -0,0 +1,55 @@
+import { Application } from 'stimulus' ;
+import TalkToUsController from 'talk-to-us_controller.js';
+
+describe('TalkToUsController', () => {
+
+    document.body.innerHTML = 
+    `
+    <section id="talk-to-us" class="talk-to-us" data-controller="talk-to-us">
+
+        <div class="talk-to-us__inner__table__column" data-target="talk-to-us.chat" id="1">
+            <p>
+                <b>Chat online</b> <br/>
+                If you have questions about getting into teaching, we can help you get the answers you need with our one-to-one live online chat service, 
```

Comment:
Might be worth removing the copy/unnecessary tags from the HTML fixture to make it easy to read